tests/sav/nitsmells_args1.res

```diff
@@ -0,0 +1,18 @@
+Test ModelBuilder
+*** CODE SMELLS METRICS ***
+--- Code Smells Metrics ---
+-------------------
+Class: Character
+Feature envy :  true
+	Affected method:
+		-quit
+Long method:  Average 1 lines
+	Affected method:
+		-total_strengh has 2 lines
```

Comment:
Well, the example itself shows how reliable this metric is... :/

I'm not convinced that a 2-lines method can really be viewed as large. Even if the average is 1 line.